### PR TITLE
Doc | Release notes v6.0 preview1

### PR DESCRIPTION
**Pull request by DavoudEshtehari**

### CODE REVIEW

**Review by arellegue — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,57 @@ All notable changes to this project will be documented in this file.
...
+This update brings the below changes over the previous release:
+
+### Breaking Changes
+
+- Removed support for .NET Standard. [#2386](https://github.com/dotnet/SqlClient/pull/2386)
+- Removed UWP (uap) references. [#2483](https://github.com/dotnet/SqlClient/pull/2483)
+
+### Added
+
+- Added `TokenCredential` object to take advantage of token caching in . [#2380](https://github.com/dotnet/SqlClient/pull/2380)
```
Comment (ErikEJ) [inline CHANGELOG.md:18]: In what??

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,57 @@ All notable changes to this project will be documented in this file.
...
+### Fixed
+
+- Fixed `Socket.Connect` timeout issue caused by thread starvation. [#2777](https://github.com/dotnet/SqlClient/pull/2777)
+- Fixed pending data with `SqlDataReader` against an encrypted column. [#2618](https://github.com/dotnet/SqlClient/pull/2618)
+- Fixed Entra authentication when using infinite connetion timeout in `ActiveDirectoryAuthenticationProvider`. [#2651](https://github.com/dotnet/SqlClient/pull/2651)
+- Fixed `GetSchema` by excluding unsupported engines due to lack of support for `ASSEMBLYPROPERTY` function. [#2593](https://github.com/dotnet/SqlClient/pull/2593)
+- Fixed SSPI retry negotiation with default port in .NET. [#2559](https://github.com/dotnet/SqlClient/pull/2559)
+- Fixed assembly path in .NET 8.0 and `.AssemblyAttributes`. [#2550](https://github.com/dotnet/SqlClient/pull/2550)
+- Fixed certificate chain validation. [#2487](https://github.com/dotnet/SqlClient/pull/2487)
+- Fixed clone of `SqlConnection` and included `AccessTokenCallback`. [#2525](https://github.com/dotnet/SqlClient/pull/2525)
```
Comment (ErikEJ) [inline CHANGELOG.md:32]: to include AccessTokenCallback

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,57 @@ All notable changes to this project will be documented in this file.
...
...net/SqlClient/pull/2369), [#2381](https://github.com/dotnet/SqlClient/pull/2381), [#2390](https://github.com/dotnet/SqlClient/pull/2390), [#2392](https://github.com/dotnet/SqlClient/pull/2392), [#2403](https://github.com/dotnet/SqlClient/pull/2403), [#2410](https://github.com/dotnet/SqlClient/pull/2410), [#2413](https://github.com/dotnet/SqlClient/pull/2413), [#2425](https://github.com/dotnet/SqlClient/pull/2425), [#2428](https://github.com/dotnet/SqlClient/pull/2428), [#2440](https://github.com/dotnet/SqlClient/pull/2440), [#2443](https://github.com/dotnet/SqlClient/pull/2443), [#2450](https://github.com/dotnet/SqlClient/pull/2450), [#2466](https://github.com/dotnet/SqlClient/pull/2466), [#2486](https://github.com/dotnet/SqlClient/pull/2486), [#2521](https://github.com/dotnet/SqlClient/pull/2521), [#2522](https://github.com/dotnet/SqlClient/pull/2522), [#2533](https://github.com/dotnet/SqlClient/pull/2533), [#2552](https://github.com/dotnet/SqlClient/pull/2552), [#2560](https://github.com/dotnet/SqlClient/pull/2560), [#2726](https://github.com/dotnet/SqlClient/pull/2726), [#2751](https://github.com/dotnet/SqlClient/pull/2751), [#2811](https://github.com/dotnet/SqlClient/pull/2811)
```
Comment (JRahnama) [inline CHANGELOG.md:56]: I think PR #2413 needs to be mentioned in changed part

Comment (DavoudEshtehari) [reply CHANGELOG.md:56]: I don’t think so. It didn’t have any effect on the functionality.

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,57 @@ All notable changes to this project will be documented in this file.
...
+- Replaced `System.Runtime.Caching` with `Microsoft.Extensions.Caching.Memory`. [#2493](https://github.com/dotnet/SqlClient/pull/2493)
+- Updated `EnableOptimizedParameterBinding` to only accept text mode commands. [#2417](https://github.com/dotnet/SqlClient/pull/2417)
+- Updated `Azure.Identity` version from `1.10.3` to `1.11.4`. [#2577](https://github.com/dotnet/SqlClient/pull/2577)
+- Updated `Azure.Core` version from `1.35.0` to `1.38.0`. [#2462](https://github.com/dotnet/SqlClient/pull/2462)
+- Updated `Azure.Security.KeyVault.Keys` version from `4.4.0` to `4.5.0`. [#2462](https://github.com/dotnet/SqlClient/pull/2462)
+- Updated `Microsoft.IdentityModel.JsonWebTokens` and `Microsoft.IdentityModel.Protocols.OpenIdConnect` from `6.35.0` to `7.5.0`. [#2429](https://github.com/dotnet/SqlClient/pull/2429)
+- Removed direct dependency to `Microsoft.Identity.Client` to take the transient dependecy through `Azure.Identity`. [#2577](https://github.com/dotnet/SqlClient/pull/2577)
+- Removed unneseccary references `Microsoft.Extensions.Caching.Memory` and `System.Security.Cryptography.Cng` after removing .NET Standard. [#2577](https://github.com/dotnet/SqlClient/pull/2577)
```
Comment (dauinsight) [inline CHANGELOG.md:51]: "unneseccary" should be "unnecessary"

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,57 @@ All notable changes to this project will be documented in this file.
...
+- Improved memory allocation when readr opened by `CommandBehavior.SequentialAccess` over the big string columns. [#2356](https://github.com/dotnet/SqlClient/pull/2356)
```
Comment (dauinsight) [inline CHANGELOG.md:52]: "readr" should be "reader"

**Review by dauinsight — APPROVED** [review]

**Pull request merged**